### PR TITLE
Support for FastAPI subclasses and removed routes

### PR DESCRIPTION
**Pull request by kunesj**
- Versioned app now uses the same FastAPI class as the original app.
- Routes can now be removed with `@api_version_remove` decorator.

### CODE REVIEW

**Comment by alexschimpf** [comment]
@kunesj Could you please add some tests?

**Comment by kunesj** [comment]
@alexschimpf Done. I added it to the `TestSimple`.

Also, I noticed that the `TestAdvanced` did not pass even before my changes. So, you should probably look into that.

**Comment by alexschimpf** [comment]
@kunesj Ok I'll take a look and report back.

File: fastapi_versionizer/versionizer.py [hunk 1]
```diff
@@ -151,6 +170,16 @@ async def get_versions_() -> VersionsModel:
     return versions
 
 
+def _get_unique_route_keys(route: BaseRoute) -> list[str]:
```
Comment (alexschimpf) [inline fastapi_versionizer/versionizer.py:173]: This package is supposed to be compatible with python 3.6+, so you'll need to use types from `typing`.

**Comment by alexschimpf** [comment]
@kunesj Btw I fixed the advanced tests, so you can pull in the latest code if you need to

**Review by alexschimpf — COMMENTED** [review]
May want to add a check for start version > remove version

**Comment by kunesj** [comment]
@alexschimpf All done

**Review by alexschimpf — APPROVED** [review]

**Comment by alexschimpf** [comment]
@kunesj Thanks for the work. I've pushed 0.1.4 to PyPI.

**Pull request merged**